sdk/communication/Azure.Communication.Common/tests/Azure.Communication.Common.Tests.csproj

```diff
@@ -14,6 +14,8 @@
     <Compile Include="$(AzureCoreSharedSources)ConnectionString.cs" LinkBase="Shared" />
     <Compile Include="..\..\Shared\src\HMACAuthenticationPolicy.cs" LinkBase="Shared\Communication" />
     <Compile Include="..\..\Shared\src\CommunicationBearerTokenCredential.cs" LinkBase="Shared\Communication" />
+    <Compile Include="..\..\Shared\src\AuthMethod.cs" LinkBase="Shared\Communication.Tests" />
+    <Compile Include="..\..\Shared\src\CommunicationTestEnvironment.cs" LinkBase="Shared\Communication.Tests" />
```

Comment:
these two can be left out of the Common.Tests.csproj - the project doesn't utilize them.

instead, add them to other modalities in the same manner as you did for Identity, so that they're able to build.

Reply:
Updated